internal/kafka/client_test.go

```diff
@@ -108,3 +121,136 @@ func TestNewSaramaClientConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestNewSaramaClient(t *testing.T) {
+	_, clientConfig := kafkatest.NewCluster(t)
+	client, err := NewSaramaClient(context.Background(), clientConfig)
+	require.NoError(t, err)
+	defer func() { assert.NoError(t, client.Close()) }()
```

Comment:
The defer isn't needed here, the close can be done after the check.

Reply:
Oops, thanks, probably removed some code